### PR TITLE
the version information of opm-fem is needed.

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -80,6 +80,7 @@ include (CMakeLists_files.cmake)
 macro (config_hook)
   opm_need_version_of ("dune-common")
   opm_need_version_of ("dune-istl")
+  opm_need_version_of ("dune-fem")
```
Comment (akva2) [inline CMakeLists.txt]: needs the appropriate if(), this is not a hard dependency.
```cmake
if(dune-fem_FOUND)
endif()
```

Comment (akva2) [reply CMakeLists.txt]: as you see from the passed build, it's not strictly a necessity. but if i saw that without the if i would be very confused.

Comment (GitPaean) [reply CMakeLists.txt]: Sure. Clear enough for me.

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by blattms** [comment]
I guess this is release material. Otherwise remove the milestone.

**Comment by GitPaean** [comment]
The comment is addressed and jenkins is happy. I am self-merging it now.

**Comment by blattms** [comment]
backported to release via #2105

**Pull request merged**